be/src/util/thread.cpp

```diff
@@ -174,6 +178,27 @@ void ThreadMgr::set_idle_sched(int64_t tid) {
         LOG(ERROR) << "set_thread_idle_sched";
     }
 }
+
+void ThreadMgr::set_low_priority(int64_t tid) {
+    if (tid == getpid()) {
+        return;
+    }
+    // From Linux kernel:
+    // In the current implementation, each unit of difference in the nice values of two
+    // processes results in a factor of 1.25 in the degree to which the
+    // scheduler favors the higher priority process.  This causes very
+    // low nice values (+19) to truly provide little CPU to a process
+    // whenever there is any other higher priority load on the system,
+    // and makes high nice values (-20) deliver most of the CPU to
+    // applications that require it (e.g., some audio applications).
+
+    // Choose 5 as lower priority value, default is 0
+    constexpr static int s_low_priority_nice_value = 5;
```

Comment:
use scan_thread_nice_value instead of fixed value

Reply:
ok